File: cookbook/java/getTargetFromUnicity.java
import java.io.FileInputStream;
import java.io.IOException;


import java.util.Map;
import java.util.Properties;

import Utils.Request;

public class getTargetFromUnicity
{
	public static void main(String[] args) throws IOException
	{
		/*
			Renseignez ici les paramètres que vous souhaitez
			@var unicity		=> le paramètre d'unicité que vous utilisez (ici l'e-mail)
		*/
		String unicity = "test@test.com";


		Properties properties = new Properties();

		try {
			properties.load(new FileInputStream("config.properties"));
		} catch (IOException e) {
			System.out.print(e.getMessage());
		}


		String xKey = properties.getProperty("xKey");
		String baseUrl = properties.getProperty("url");
		String url = baseUrl + "targets?unicity=" + unicity;

		Map<String, String> resp = null;
		resp = Request.connection(url, xKey, null, "GET");

		// Vérification de la réponse
		if (resp.get("code").equals("200")) {
			// Affichage des données
			System.out.print(resp.get("reply"));
		}
		else {
			// Affichage de l'erreur
			System.out.print("Error : " + resp.get("code") + " " + resp.get("mess"));
		}
	}
}

File: cookbook/Csharp/getTargetFromUnicity.cs
using System;
using System.IO;
using System.Net;

namespace getTargetFromUnicity
{
    class Program
    {
        static void Main(string[] args)
        {
            //Ici, renseignez l'email dont vous voulez obtenir les valeurs des champs et la X-Key
            string unicity = "test@test.com";
            string xKey = "ABCDEFJHIJKLMNOPQRSTUVWXYZ0123456789";
            
            //Lancement de la connection pour remplir la requete
            string url = "http://v8.mailperformance.com/targets?unicity=" + unicity;
            HttpWebRequest con = (HttpWebRequest)WebRequest.Create(url);
            con.Method = "GET";

            //Mise en place du xKey et des options
            con.Headers.Add("X-Key", xKey);
            con.ContentType = "application/json";

            //Verification des reponses
            HttpWebResponse httpResponse = (HttpWebResponse)con.GetResponse();
            if ((int)httpResponse.StatusCode != 200)
            {
                //Affichage de l'erreur
                Console.Write("Error : {0} {0}", (int)httpResponse.StatusCode, httpResponse.StatusCode.ToString());
            }
            else
            {
                //Lecture des donnees
                StreamReader reader = new StreamReader(con.GetResponse().GetResponseStream());
                string responseString = reader.ReadToEnd();
                Console.Write("{0}", responseString);
            }

            //Attente de lecture (optionnel)
            Console.ReadLine();

            httpResponse.Close();
        }
    }
}
